tools/apiview/parsers/cpp-api-parser/ci.yml

```diff
@@ -118,12 +119,32 @@ stages:
                   - checkout: none
                   - download: current
                     artifact: release-artifacts
-                  - task: UniversalPackages@0
-                    displayName: Publish ApiView Parser.
+                  - task: AzurePowerShell@5
+                    displayName: Publish Public ApiView Parser
                     inputs:
-                      command: publish
-                      publishDirectory: $(Pipeline.Workspace)/release-artifacts
-                      vstsFeedPublish: 'internal/AzureSDKForCpp'
-                      vstsFeedPackagePublish: 'parseazuresdkcpp'
-                      versionOption: minor
-                      packagePublishDescription: 'Azure C++ SDK API View Parser.'
+                      azureSubscription: 'Azure SDK Artifacts'
+                      ScriptType: 'InlineScript'
+                      azurePowerShellVersion: LatestVersion 
+                      pwsh: true
+                      Inline: |
+                        azcopy copy "$(Pipeline.Workspace)/release-artifacts/*" "https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-cpp/parseazuresdkcpp" --recursive
```

Comment:
Let's add the version to the path here so that we can reference a specific version when we are trying to consume it and we can have side-by-side versions as needed.